docs/UserGuide.md

```diff
@@ -101,35 +101,28 @@ For example, 7:00 AM is `0700` while 7:00 PM is `1900`.
 Format: `add n/NAME_OF_ASSIGNMENT d/DEADLINE_OF_ASSIGNMENT mod/MODULE_CODE_OF_ASSIGNMENT [p/PRIORITY_OF_ASSIGNMENT] 
 [remind]`
 
-You can add your assignments into your schedule so that you can manage all your assignments and
- academic tasks conveniently. 
+You can add assignments into your assignment list to manage and keep track of all your academic tasks. 
 
-<div markdown="span" class="alert alert-success">
-**:bulb: Tip:**
-You can include `remind` when adding an assignment instead of using the `remind` command to set reminders after adding an assignment.
-</div>
-
-For example, to add a lab report assignment that is **due** on 23th April 2020 12.30pm and **associated** with the module CS2100, you can 
-simply enter `add n/Lab report d/23-04-2020 1230 mod/CS2100`. All the information of the assignment will be displayed as shown in the labelled diagram below.
+For example, to add a lab report assignment that is **due on 23th April 2020 12.30pm** and **associated with the module CS2100**, you can 
+simply enter `add n/Lab report d/23-04-2020 1230 mod/CS2100`. Once the assignment is added, it will appear in the assignment list and all its information will be displayed as shown in the figure below.
 
    ![UserGuideAddCommand](images/UserGuideAddCommand.PNG)
    *Figure 3: Information of the assignment added labelled*
 
 <div style="page-break-after: always;"></div>
 
+<div markdown="span" class="alert alert-success">
+**:bulb: Tip:**
+If the lab report assignment is very urgent, you can attach a **high priority tag** to it and enter `add n/Lab report d/23-04-2020 1230 mod/CS2100 p/HIGH`.
+</div> 
```

Comment:
The phrasing currently sounds as if you tag the assignment with a priority tag THEN you enter the add command to add an assignment, rather than using the add command to add a priority tag while adding an assignment.

Maybe phrasing it as "... you can attach a high priority tag to it **by entering** ...." would be clearer

Reply:
changed, thank you!